support-frontend/assets/components/footerCompliant/footer.jsx

```diff
@@ -50,11 +50,30 @@ function Footer({
               {disclaimer && <ContribLegal countryGroupId={countryGroupId} />}
             </Rows>
           </div>
-        </Content>
+        </FooterContent>
       }
-      <Content border appearance={appearance}>
+      <FooterContent border>
+        <ul css={linksList}>
+          <li css={link}>
+            <a href="https://www.theguardian.com/help/privacy-policy">Privacy Policy</a>
+          </li>
+          <li css={link}>
+            <a href="https://www.theguardian.com/help/contact-us">Contact us</a>
+          </li>
+          <li css={link}>
+            <a href="https://www.theguardian.com/subscriber-direct/subscription-frequently-asked-questions">FAQs</a>
+          </li>
+          <li css={link}>
+            <a href="https://www.theguardian.com/info/2014/aug/06/guardian-observer-digital-subscriptions-terms-conditions">Terms & Conditions</a>
```

Comment:
This url is for the Digital subscription only, do we need to parameterise this for use with other products?